src/Domain/Creators/Cylinder.cpp

```diff
@@ -25,6 +26,185 @@ struct Inertial;
 }  // namespace Frame
 
 namespace domain {
+namespace partitioning_helpers {
+
+std::vector<
+    std::unique_ptr<CoordinateMapBase<Frame::Logical, Frame::Inertial, 3>>>
+list_blocks(const double inner_radius, const double outer_radius,
```

Comment:
I would rename this `cyl_wedge_coordinate_maps` or similar, this is similar to `wedge_coordinate_maps` in line 607 of Domain/DomainHelpers (we can move this there and think of renaming the other to `sph_wedge_coordinate_maps`)